server/docHandlers.js

```diff
@@ -0,0 +1,31 @@
+const Doc = require('doc');
+const Taboo = require('taboo');
+
+const savedoc = (req, res) => {
```

Comment:
Since this handles a request, it should send a response, otherwise the request will timeout

Reply:
sends message now